lib/Dependency.js

```diff
@@ -7,7 +7,7 @@ function Dependency() {
 }
 module.exports = Dependency;
 
-Dependency.prototype.isEqualResource = function( /* other */ ) {
```

Comment:
An alternative could be to just receive the parameter anyway and not use it. `no-unused-vars` is disabled here anyway, so there would be nothing to complain about it :cry:

Even if `no-unused-vars` were to be enabled, something like `{ "argsIgnorePattern": "^_" }` could be used to allow prefixing the argument name with a `_` to avoid the warning.